.github/workflows/conda_ci_container_cicd.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Publish Conda CI Images
+on:
+  schedule:
+    - cron:  '0 0 * * 0'
+  pull_request:
+    paths:
+      - 'ci/Dockerfile'
+      - '.github/workflows/conda_ci_container_cicd.yml'
+  push:
+    paths:
+      - 'ci/Dockerfile'
+      - '.github/workflows/conda_ci_container_cicd.yml'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.7, 3.8, 3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login to DockerHub
```

Comment:
```suggestion
      - name: Login to GitHub Container Registry
```

Reply:
I was wondering if this separate section is needed if the `docker/build-push-action` action also supports auth parameters out of the box? Or is this because of using buildx?

Reply:
ah. that can be maybe simplified... also we don't need buildx... lets see